featurebyte/service/online_serving.py

```diff
@@ -201,56 +289,20 @@ async def get_request_code_template(  # pylint: disable=too-many-locals
                 column for column in table.columns_info if column.entity_id in entities
             ]
             if entity_columns:
-                graph, node = table.construct_graph_and_node(
-                    feature_store_details=feature_store.get_feature_store_details(),
-                    table_data_dict=table.dict(by_alias=True),
-                )
-                sample_data = dataframe_from_json(
-                    await self.preview_service.preview(
-                        preview=FeatureStorePreview(
-                            feature_store_name=feature_store.name,
-                            graph=graph,
-                            node_name=node.name,
-                        ),
-                        limit=num_rows,
-                    )
-                )
                 for column in entity_columns:
-                    entities[column.entity_id]["sample_value"] = sample_data[column.name].to_list()
-
-        entity_serving_names = json.dumps(
-            [
-                {
-                    entity["serving_name"][0]: entity["sample_value"][row_idx]
-                    for entity in entities.values()
-                }
-                for row_idx in range(num_rows)
-            ]
-        )
-
-        # construct serving url
-        headers = {
-            "Content-Type": "application/json",
-            "active-catalog-id": str(feature_list.catalog_id),
-            "Authorization": "Bearer <API_TOKEN>",
-        }
-
-        # populate template
-        with open(
-            file=template_file_path,
-            mode="r",
-            encoding="utf-8",
-        ) as file_object:
-            template = Template(file_object.read())
+                    entities[column.entity_id][
```

Comment:
If an entity appear in multiple tables, will this repeat the extraction of unique values from those tables?

Reply:
yes, is there any way to decide which table should be used in such cases?

Reply:
If the purpose is for getting unique values, I think it makes sense to use the table where the entity is the primary key / natural key if available.